lib/client-assets.php

```diff
@@ -66,6 +66,28 @@ function gutenberg_get_script_polyfill( $tests ) {
 	return $polyfill;
 }
 
+/**
+ * Registers the main TinyMCE scripts.
+ */
+function register_tinymce_scripts() {
+	global $tinymce_version, $concatenate_scripts, $compress_scripts;
+	if ( ! isset( $concatenate_scripts ) ) {
+		script_concat_settings();
+	}
+	$suffix     = SCRIPT_DEBUG ? '' : '.min';
+	$compressed = $compress_scripts && $concatenate_scripts && isset( $_SERVER['HTTP_ACCEPT_ENCODING'] )
+		&& false !== stripos( $_SERVER['HTTP_ACCEPT_ENCODING'], 'gzip' );
+	// Load tinymce.js when running from /src, otherwise load wp-tinymce.js.gz (in production) or
```

Comment:
I'm not sure I understand what the core version has to do with this. Shouldn't we register the unminified file when Gutenberg is in development mode, not core?

Reply:
I just wanted to mimic exactly what core does, so we end up loading the exact same URL in Gutenberg and in Core no matter the setup (`src` core with gutenberg for instance) . That way we deduplicate the loading of the script and ensure TinyMCE is loaded in the exact same conditions (use `wp-tinymce.php` when Core does so or use the `js` files)

Reply:
Okay, makes sense